tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -328,10 +342,9 @@ kbs_k8s_svc_host() {
 	if kubectl get ingress -n "$KBS_NS" 2>/dev/null | grep -q kbs; then
 		kubectl get ingress kbs -n "$KBS_NS" \
 			-o jsonpath='{.spec.rules[0].host}' 2>/dev/null
-	elif kubectl get svc kbs-nodeport -n "$KBS_NS" &>/dev/null; then
+	elif kubectl get svc kbs -n "$KBS_NS" &>/dev/null; then
 			local host
-			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n "$KBS_NS")
-			[ -z "$host"] && host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}' -n "$KBS_NS")
+			host=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[0].address}')
```

Comment:
This works well for me, but I remember using the `ExternalIP` type was suppose to be safer as with multiple addresses it should attempt to get the public facing one. The first address might be anything (even localhost).

Reply:
@ldoktor when using `ExternalIP` it does not bring anything
```
$ kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}' -n coco-tenant
```
Only `InternalIP` brings the address
```
$ kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}' -n coco-tenant
10.0.0.5
```

Reply:
This is from stack-overflow, when I was searching on how to properly get the right public-facing interface. It made sense although I only tested it with kcli which only comes with InternalIP. I assumed other flavours might expose the expected one (or one has to expose it themselves, if needed). This way we'd reflect the configuration so it looked safe. Anyway I don't have a strong opinion about this, I simply don't know which is correct (but asking for InternalIP seems odd as we want a publicly available interface so either simply trust the first interface would be fine or do External followed by Internal if unavailable)

Reply:
@ldoktor thanks for the feedback me neither I am not an expert....